prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -58,7 +58,7 @@ void efa_rdm_txe_construct(struct efa_rdm_ope *txe,
 	txe->cq_entry.buf = OFI_LIKELY(txe->cq_entry.len > 0) ? txe->iov[0].iov_base : NULL;
 
 	if (ep->user_info->mode & FI_MSG_PREFIX) {
-		ofi_consume_iov(txe->iov, &txe->iov_count, ep->msg_prefix_size);
+		ofi_consume_iov_desc(txe->iov, txe->desc, &txe->iov_count, ep->msg_prefix_size);
```

Comment:
A further improvement on this should be constructing the pkt entry just from txe->iov[0].iov_base. Then we don't need to walk the iov here. Can be a follow-up PR